iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/telemetry/ReceiveMessagesTests.java

```diff
@@ -93,81 +93,4 @@ else if (testInstance.identity.getClient() instanceof ModuleClient)
         Thread.sleep(200);
         testInstance.identity.getClient().closeNow();
     }
-
-    @Test
-    @StandardTierHubOnlyTest
-    public void receiveBackToBackUniqueC2DCommandsOverAmqpsUsingSendAsync() throws Exception
```

Comment:
Redundant test. The other test in this suite already ensures that c2d messages can be received over AMQP and AMQP_WS